{
  try {
    releaseLock();
  }
 catch (  Exception e) {
    logger.error("Got an exception during close()",e);
  }
  this.fileSystem.close();
}
